homeassistant/components/ecobee/binary_sensor.py

```diff
@@ -4,7 +4,7 @@
     DEVICE_CLASS_OCCUPANCY,
 )
 
-from .const import DOMAIN
+from .const import DOMAIN, ECOBEE_MODEL_TO_NAME, MANUFACTURER, _LOGGER
```

Comment:
Ideally, `ECOBEE_MODEL_TO_NAME` should be a part of `python-ecobee-api` as it is API-specific; I intend to eventually move it there as part of a future planned refactoring of parts of `python-ecobee-api`.